main.cpp

```diff
@@ -1,8 +1,14 @@
 #include <iostream>
+#include <cmath>
 
 #include "create_tridiagonal.hpp"
 #include "matrix_operations.hpp"
+#include "test_matrix.hpp"
 
-int main(int argc, char *argv[]) {
+using namespace std;
+
+int main(int argc, char *argv[]) 
+{
+    test_6_by_6_symmetric_with_eig_sym();
     return 0;
 }
```

Comment:
```suggestion
}

```
La til newline på slutten av filen.